.travis.yml

```diff
@@ -11,5 +11,33 @@ cache:
   directories:
   - $HOME/.m2
 
-script:
-  - ./extras/travis_push_docker_hub.sh
+  
+stages:
+  - name: unit test
+    if: type IN (push, pull_request)
+  - name: integ test
+    if: type IN (push, pull_request)
+  - name: image build
+    if: type = push AND branch = master       
+    
+jobs:
+  include:
+    - stage: unit test
+      script: mvn test
+    - stage: integ test
+      before_script:
+        - sudo service postgresql stop
+        - cd extras/postgresdb
+        - docker build -t ega-data-api/ega-postgres .
```

Comment:
Why do we build this image on Travis? Is it possible to build it once and just publish it to Docker Hub?

Reply:
Yes, we can do that but if there is some schema change then that image has to be updated which is a manual task. So I thought of creating a fresh Postgres db with init.sql every time.

Reply:
I see. I was confused because in LocalEGA repo we now are using external DB image, which lives in its own repo. But in this case it's OK, I guess.